config-v3-documentation

```diff
@@ -62,6 +62,7 @@ checker=check
 out_check=judge
 # Describes how to check outputs
 #   - diff: checks that contestant output equals correct output (ignores whitespace) (default)
+#   - tokens: uses a fast, versatile file equality checker (ignores whitespace)
```

Comment:
Isn't tokens a better default?

Reply:
Probably. I wanted to keep the default unchanged for backwards compatibility, but that might not be necessary. There aren't any Kasiopea tasks that doesn't set `out_check` explicitly.

Reply:
I was thinking about adding it to update script

Reply:
What do you want to add to the update script? Adding `out_check=diff` if `out_check` isn't specified?

Reply:
Yup

Reply:
Turns out the documentation was wrong: The `out_check` key has no default and is mandatory. I've fixed the documentation.